mygrad/tensor_base.py

```diff
@@ -277,7 +277,7 @@ def creator(self):
 
     def __len__(self):
         return len(self.data)
-    
```

Comment:
In the future, if you accidentally "touch" files like this, just do `git checkout -- <file>` to undo the modifications. This will ensure that only the relevant files are affected by your PR.

Reply:
I can still revert these changes since they're isolated to `tensor_base.py` and I removed all the other changes I previously had in that file. Want me to undo these?